src/app/core/util/utils.ts

```diff
@@ -8,6 +8,11 @@ export class Amount {
     return this.amount;
   }
 
+  public getAmountWithFee(fee: number) {
```

Comment:
This doesn't belong in the Amount model, move it elsewhere

Reply:
I don't think we need to move it elsewhere, its belongs in Amount we calculating total amount with a fee.

Reply:
Amount are just amounts.
A fee is not part of any amount so it doesn't belong.

If I have an amount 5 apples, why would I do getAmountWithFee?
Anyways, an Amount doesn't necessarily have to be part of a tx.